src/zcash/address/zip32.h

```diff
@@ -152,17 +159,20 @@ class SaplingDiversifiableFullViewingKey {
     // Returns the first index starting from j that generates a valid
     // payment address, along with the corresponding address. Throws
     // a runtime error if the diversifier space is exhausted.
-    std::pair<SaplingPaymentAddress, diversifier_index_t> FindAddress(diversifier_index_t j) const {
+    std::pair<SaplingPaymentAddress, diversifier_index_t> FindAddress(diversifier_index_t j) const
+    {
         auto addr = Address(j);
         while (!addr.has_value()) {
-            if (!j.increment())
-                throw std::runtime_error(std::string(__func__) + ": diversifier index overflow.");;
+            if (!j.increment()) {
+                throw std::runtime_error(std::string(__func__) + ": diversifier index overflow.");
+            };
```

Comment:
https://github.com/zcash/zcash/pull/6641/commits/8519bca382110eafc2246483391846f483be5225 will force getting rid of the extraneous `;`.